src/battle/area_arn/4FF0D0.c

```diff
@@ -2,4 +2,25 @@
 
 #define NAMESPACE b_area_arn
 
-INCLUDE_ASM(s32, "battle/area_arn/4FF0D0", func_80218680_4FF0D0);
+ApiStatus func_80218680_4FF0D0(Evt* script, s32 isInitialCall) {
+    Bytecode *readPos;
+    s32 var1;
+    s32 var2;
+    s32 var3;
+    f32 var4;
+    Temp8025D160* temp8025D160;
+
+    readPos = script->ptrReadPos;
```

Comment:
We call `readPos` `args`, and you can initialize it on line 6 where it's declared too. You can the initialize the temp struct pointer as well on the following line, and vars 1-4 on the following lines where they're declared